include/sound/sof/dai-intel.h

```diff
@@ -49,6 +49,9 @@
 /* bclk idle */
 #define SOF_DAI_INTEL_SSP_CLKCTRL_BCLK_IDLE_HIGH	BIT(5)
 
+/* DMIC max. four controllers for eight microphone channels */
```

Comment:
nit on commit title; Can we use ASoC: SOF: Intel: to show this is not a core change but an Intel specific one?

Reply:
This header file change is for Intel only but the change in topology to remove the capability for variable length array is common. I didn't originally split these because git tree with only one of the changes would cause failed functionality.

Should I do the split or just say this is Intel specific change. No other than Intel SOF platforms today use this load code for DMIC DAI.

Reply:
it's not really common, the dmic load is only for Intel. it's fine to keep the code as is. just change the commit title

Reply:
I just tried my luck by just changing title of second commit with less effort option. Please vote if OK or not!

Reply:
it's fine